packages/handlefi/src/utils.ts

```diff
@@ -0,0 +1,79 @@
+import type { ActionParams, FilterOperator } from '@rabbitholegg/questdk'
+import { getAddress, type Address, type Hash } from 'viem'
+
+export enum Chains {
```

Comment:
Is this used anywhere?

Reply:
Removed this since we already have a constant for arbitrum chain id.